packages/cli-platform-android/src/config/__fixtures__/files/AndroidManifest-className.xml

```diff
@@ -3,6 +3,9 @@
 
 	<application android:name=".MainApplication">
 		<activity android:name="com.example.ExampleAppActivity">
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+      </intent-filter>
```

Comment:
Can we align this?

Reply:
https://github.com/react-native-community/cli/pull/2432/commits/f5ff1fd33bf7140f7ed9c2a5d4c820f335f7e644 there was something wrong with spaces, my editor was showing it correctly 🙈